lib/kafo/kafo_configure.rb

```diff
@@ -371,6 +369,11 @@ def validate_all(logging = true)
 
     def run_installation
       self.class.hooking.execute(:pre)
+
+      hiera = HieraConfigurer.new(config.app[:hiera_config], config.modules, config.app[:order])
+      hiera.write_configs
```

Comment:
Shouldn't this respect `noop?`? I suppose it would fail if `kafo_answers` source would be missing but still, noop shouldn't do any changes I think. If hiera config would be right it would work without write_configs I think.